src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -286,7 +287,7 @@ function BaseTextInput(
                     style={[
                         autoGrowHeight && styles.autoGrowHeightInputContainer(textInputHeight, variables.componentSizeLarge, typeof maxHeight === 'number' ? maxHeight : 0),
                         !isMultiline && styles.componentHeightLarge,
-                        containerStyles,
+                        touchableInputWrapperStyle,
```

Comment:
Forgot to copy the changes in native file, which caused https://github.com/Expensify/App/issues/34136